ivy/functional/backends/tensorflow/manipulation.py

```diff
@@ -231,3 +232,10 @@ def clip(
     else:
         ret = tf.clip_by_value(x, x_min, x_max)
     return ret
+
+
+def flatten(x):
+    shape = 1
+    for dim in x.shape:
+        shape *= dim
+    return tf.reshape(x, shape)
```

Comment:
Does TensorFlow not support `tf.reshape(x, -1)`?

Reply:
If it does not, you can do this in one line like so:

```
import math
tf.reshape(x, math.prod(x.shape))
```

Reply:
yh I will change this.